x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/datascience/DataScienceFeatureSetUsage.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.core.datascience;
+
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.xpack.core.XPackFeatureSet;
+import org.elasticsearch.xpack.core.XPackField;
+
+import java.io.IOException;
+import java.util.Objects;
+
+public class DataScienceFeatureSetUsage extends XPackFeatureSet.Usage {
+
+    public DataScienceFeatureSetUsage(boolean available, boolean enabled) {
+        super(XPackField.DATA_SCIENCE, available, enabled);
+    }
+
+    public DataScienceFeatureSetUsage(StreamInput input) throws IOException {
+        super(input);
+    }
+
+    @Override
+    public void writeTo(StreamOutput out) throws IOException {
+        super.writeTo(out);
```

Comment:
Why override this at all if it's just going to call it's super method?

Reply:
Oops, sorry, this is WIP at the moment.  I'm adding stats/telemetry and this was a convenient checkpoint to commit (the boilerplate before the actual stats were implemented).

I'll give you a ping once the rest of the telemetry stuff is done :)